x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/details_step/details_step_form.tsx

```diff
@@ -266,22 +269,46 @@ export const DetailsStepForm: FC<CreateAnalyticsStepProps> = ({
           />
         </EuiFormRow>
       )}
-      <EuiFormRow
-        label={i18n.translate('xpack.ml.dataframe.analytics.create.resultsFieldLabel', {
-          defaultMessage: 'Results field',
-        })}
-        helpText={i18n.translate('xpack.ml.dataframe.analytics.create.resultsFieldHelpText', {
-          defaultMessage:
-            'Defines the name of the field in which to store the results of the analysis. Defaults to ml.',
-        })}
-      >
-        <EuiFieldText
+      <EuiFormRow fullWidth>
+        <EuiSwitch
```

Comment:
When I clone a job which is using the default results field of `ml`, this switch is set to off. I expected it to be on, with the results fields text field hidden:

![image](https://user-images.githubusercontent.com/7405507/94803385-1f255f80-03e1-11eb-9e3f-84c7e0223842.png)

Can it to be set to 'On' when cloning a job which is using the default of `ml`?

Reply:
Good point! Updated in fe70cd94ec5931231b7ac4b08ec7f2c7a1f021b8